lib/index.d.ts

```diff
@@ -0,0 +1 @@
+// TODO: Write type for standard librairy
```

Comment:
:+1:  I think in it would be better if we eventually move completely to typescript and generate JS from TS source.

Reply:
Ok.